### PR TITLE
add test for MathJax/MathJax#912

### PR DESCRIPTION
**Pull request by pkra**

### CODE REVIEW

**Comment by pkra** [comment]
Not sure if that's what you had in mind. Should I split it into two tests?

File: testsuite/UI/issue912.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        var jax = MathJax.Hub.getAllJax()[0];
+        var mmlString = jax.root.toMathML('',jax);
+        var span = document.createElement('span');
+        span.innerHTML = mmlString;
+        var mstyle = span.firstChild.getElementsByTagName('mstyle')[0];
+        var check;
+        try {
+          check = ( mstyle.getAttribute('scriptlevel') === '0' && mstyle.getAttribute('displaystyle') === 'false');
+          }
+        catch (err) {check = false};
```
Comment (dpvc) [inline testsuite/UI/issue912.html]: This works, but you should be able to do it without the overhead of `try/catch`.  How about:

``` javascript
var check = !!(mstyle && mstyle.getAttribute('scriptlevel') === '0' && mstyle.getAttribute('displaystyle') === 'false');
```

**Comment by dpvc** [comment]
I think one test is fine for this.  If you want, you could do

``` javascript
        return [
          newScriptReftestResult("Scriplevel 0",(mstyle && mstyle.getAttribute('scriptlevel') === '0')),
          newScriptReftestResult("Displaystyle false",(mstyle && mstyle.getAttribute('displaystyle') === 'false'))
        ];
```

which would be one test file with each part tested separately.

File: testsuite/UI/issue912.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        return [
+          newScriptReftestResult("Check displaystyle and scriptlevel changes in toMathML() output",
+            check),
```
Comment (dpvc) [inline testsuite/UI/issue912.html]: You don't want the final comma, since that will fail in IE < 9.

**Comment by pkra** [comment]
How about now?

**Comment by dpvc** [comment]
Looks good.  The copyright date could be changed and scriptlevel and displaystyle shouldn't be capitalized now that they are not the beginning of the sentence, but I wouldn't worry about those.  I'll merge...

**Pull request merged**